lib/wrapbox/runner/ecs.rb

```diff
@@ -398,7 +404,7 @@ def register_task_definition(container_definition_overrides)
             task_role_arn: task_role_arn,
             execution_role_arn: execution_role_arn
           }).task_definition
-        rescue Aws::ECS::Errors::ClientException
+        rescue Aws::ECS::Errors::ClientException, Aws::ECS::Errors::ThrottlingException
```

Comment:
[imo] I think this change are not required because `register_task_definition` is called only once.

Reply:
I've fixed.